beaker-rspec.gemspec

```
@@ -5,14 +5,14 @@ require 'beaker-rspec/version'
 Gem::Specification.new do |s|
   s.name        = "beaker-rspec"
   s.version     = BeakerRSpec::Version::STRING
-  s.authors     = ["Puppetlabs"]
-  s.email       = ["sqa@puppetlabs.com"]
-  s.homepage    = "https://github.com/puppetlabs/beaker-rspec"
+  s.authors     = ["Vox Pupuli"]
+  s.email       = ["voxpupuli@groups.io"]
+  s.homepage    = "https://github.com/voxpupuli/beaker-rspec"
   s.summary     = %q{RSpec bindings for beaker}
-  s.description = %q{RSpec bindings for beaker, see https://github.com/puppetlabs/beaker}
+  s.description = %q{RSpec bindings for beaker, see https://github.com/voxpupuli/beaker}
   s.license     = 'Apache-2.0'
 
-  s.required_ruby_version = Gem::Requirement.new('>= 2.1.8', '<4.0.0')
+  s.required_ruby_version = '>= 2.4.0'
```

I'd be in favor of keeping the upper bound.
```suggestion
  s.required_ruby_version = '>= 2.4.0', '<4.0.0'
```

---

I went with `'>= 2.4.0'` because we've that in other modules as well, but we can keep the old style.